src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -44,6 +44,17 @@ def register_options(cls, register):
                 "Infer a target's imported dependencies by parsing import statements from sources."
             ),
         )
+        register(
+            "--string-imports",
+            default=False,
+            type=bool,
+            help=(
+                "Infer a target's dependencies based on strings that look like dynamic imports, "
+                "such as `importlib.import_module('example.subdir.Foo')`. This can be useful if "
```

Comment:
Not sure this is a good example, because we don't actually look for `importlib.import_module` AFAICR? And the Django example certainly doesn't. I think it's best to talk about `dynamic dependences` and avoid the word `import`.

Reply:
Yeah, we only look for `example.subdir.Foo`.